league/manager/models.py

```diff
@@ -46,33 +48,37 @@ def league_winner(self):
 
 
 class Team(models.Model):
-    league = models.ForeignKey(League, on_delete=models.CASCADE)
+    team_league = models.ForeignKey(League, on_delete=models.CASCADE)
```

Comment:
What about teams_league? I think that it would be easier to understand.

Reply:
team_league is already fine enough